README.md

```diff
@@ -1,2 +1,149 @@
-# Yoga-Class-Refactored
-A Refactored Yoga Application
+# About
+- Backend application demo using Spring Boot
+- Application to manage Yoga classes
+- Refactored of [old version](https://github.com/web-development-course-way/Yoga-Class)
+# Tech Stack
+- Docker
+- PostgreSQL & pgAdmin 
+- Java & Spring boot
+# Installation
+01. Clone Repository and open project in IDE
+02. [Set up database](https://github.com/web-development-course-way/Yoga-Class-Refactored/tree/enhancement/docs?tab=readme-ov-file#setting-up-database)
```

Comment:
remove the hyperlink and instead says it will be discusses in another section, to avoid jumping back and forth.

SPECIALLY when the "another section" is the literal next section 😄